website/docs/components/stepper/list/partials/specifications/states.md

```diff
@@ -0,0 +1,3 @@
+## States
+
+![Stepper list states](/assets/components/stepper/list/stepper-list-states.png =811x*)
```

Comment:
The images should be appropriately sized in Figma before export that you shouldn't need "=811x*"